application/gqlgen/thread.go

```diff
@@ -72,6 +72,23 @@ func (r *threadResolver) Post(ctx context.Context, obj *models.Thread) (*models.
 	return obj.GetPost(selectedFields)
 }
 
+func (r *threadResolver) UnreadMessageCount(ctx context.Context, obj *models.Thread) (int, error) {
+	if obj == nil {
+		return 0, nil
+	}
+	lastViewedAt, err := obj.GetLastViewedAt(models.GetCurrentUserFromGqlContext(ctx, TestUser))
+	if err != nil {
```

Comment:
```suggestion
	if err != nil || lastViewedAt == nil {
```

Reply:
Good catch.  I've added a separate `if` block for it now to log a different message.